pydicom/config.py

```diff
@@ -385,3 +390,52 @@ def debug(debug_on=True, default_handler=True):
 
 # force level=WARNING, in case logging default is set differently (issue 103)
 debug(False, False)
+
+_use_future = False
+_use_future_env = os.getenv("PYDICOM_FUTURE")
+
+if _use_future_env:
+    if _use_future_env.lower() in ["true", "yes", "on", "1"]:
+        _use_future = True
+    elif _use_future_env.lower() in ["false", "no", "off", "0"]:
+        _use_future = False
+    else:
+        raise ValueError(
+            "Unknown setting for environment variable "
+            "PYDICOM_FUTURE. Use True or False."
+        )
+
+
+def future_behavior(enable_future: Optional[bool] = True) -> None:
```

Comment:
You don't actually need `Optional` here, only where `None` is a possible value. It's a shortcut for `Union[..., None]`

```python
def myfunc(..., filename: Optional[str] = None) -> Optional[str]:
    return filename
```